### PR TITLE
[Enhance] Move train_cfg test_cfg to model

### PR DESCRIPTION
**Pull request by xiliu8006**
Move train_cfg/test_cfg into model in model configs

### CODE REVIEW

**Review by Tai-Wang — APPROVED** [review]

File: configs/centerpoint/centerpoint_0075voxel_second_secfpn_circlenms_4x8_cyclic_20e_nus.py [hunk 1]
```diff
@@ -1,3 +1,22 @@
 _base_ = ['./centerpoint_0075voxel_second_secfpn_4x8_cyclic_20e_nus.py']
 
-test_cfg = dict(pts=dict(nms_type='circle'))
+voxel_size = [0.075, 0.075, 0.2]
+point_cloud_range = [-54, -54, -5.0, 54, 54, 3.0]
+
+model = dict(
+    pts_voxel_layer=dict(
```
Comment (ZwwWayne) [inline configs/centerpoint/centerpoint_0075voxel_second_secfpn_circlenms_4x8_cyclic_20e_nus.py:7]: Do we need lines 7-12?

File: configs/centerpoint/centerpoint_0075voxel_second_secfpn_dcn_circlenms_4x8_cyclic_20e_nus.py [hunk 2]
```diff
@@ -1,6 +1,12 @@
 _base_ = ['./centerpoint_0075voxel_second_secfpn_4x8_cyclic_20e_nus.py']
 
+voxel_size = [0.075, 0.075, 0.2]
+point_cloud_range = [-54, -54, -5.0, 54, 54, 3.0]
+
 model = dict(
+    pts_voxel_layer=dict(
```
Comment (ZwwWayne) [inline configs/centerpoint/centerpoint_0075voxel_second_secfpn_dcn_circlenms_4x8_cyclic_20e_nus.py:7]: Do we need lines 7-9?

File: configs/centerpoint/centerpoint_01voxel_second_secfpn_circlenms_4x8_cyclic_20e_nus.py [hunk 3]
```diff
@@ -1,3 +1,10 @@
 _base_ = ['./centerpoint_01voxel_second_secfpn_4x8_cyclic_20e_nus.py']
 
-test_cfg = dict(pts=dict(nms_type='circle'))
+point_cloud_range = [-51.2, -51.2, -5.0, 51.2, 51.2, 3.0]
+
+model = dict(
+    pts_voxel_layer=dict(point_cloud_range=point_cloud_range),
```
Comment (ZwwWayne) [inline configs/centerpoint/centerpoint_01voxel_second_secfpn_circlenms_4x8_cyclic_20e_nus.py:6]: Do we need lines 6-7

**Comment by ZwwWayne** [comment]
Please also modify the documentation.

File: tools/train.py [hunk 4]
```diff
@@ -136,7 +136,7 @@ def main():
     meta['seed'] = args.seed
 
     model = build_detector(
-        cfg.model, train_cfg=cfg.train_cfg, test_cfg=cfg.test_cfg)
+        cfg.model, train_cfg=cfg.model.train_cfg, test_cfg=cfg.model.test_cfg)
```
Comment (ZwwWayne) [inline tools/train.py:139]: Can we directly use `cfg.model` without setting train_cfg and test_cfg?

**Comment by ZwwWayne** [comment]
Add migration section for the config refactoring like [here](https://github.com/open-mmlab/mmdetection/blob/master/docs/tutorials/config.md#deprecated-train_cfgtest_cfg).

**Review by ZwwWayne — CHANGES_REQUESTED** [review]

**Pull request merged**